go.mod

```diff
@@ -237,6 +237,7 @@ require (
 	github.com/samber/lo v1.38.1 // indirect
 	github.com/samber/slog-common v0.11.0 // indirect
 	github.com/skeema/knownhosts v1.2.1 // indirect
+	github.com/stretchr/objx v0.5.0 // indirect
```

Comment:
Why do we need this change? I don't see any use on the code and we didn't update any dependency

Reply:
I'm not sure, I just run go mod tidy from the vscode and it was added. I will delete it and executed again to see if it pops up again